src/Scrollable/_variables.scss

```diff
@@ -0,0 +1,33 @@
+
+// Scrollable
+
+// Padding applied to the scrollable body
+$scrollable-top-margin:                  0.46rem !default;
+$scrollable-bottom-margin:               5.4rem !default;
+$scrollable-inner-padding:               1.58rem !default;
+$scrollable-shadow-padding:              0.96rem !default;
```

Comment:
I don't think this should be built into the component. Ideally scrollable body has no padding and if padding is desired the developer would write something like:
```
<Scrollable>
  <div className="p-4">
   ...
  </div>
</Scrollable>
```

I realize this may mess with the way shadows are positioned...

Reply:
hmm ok going to try this too, my shadows were going berserk when I changed paddings ...

Reply:
yeah so I am trying to remove all the margin and padding settings, does not look like I need those anyway for the shadows to work

Reply:
also added an example to showcase page showing consumer can add their own padding, it still works the same :)